internal/daemon/cluster/handlers/worker_service.go

```diff
@@ -61,8 +61,8 @@ func (ws *workerServiceServer) Status(ctx context.Context, req *pbs.StatusReques
 
 	wStat := req.GetWorkerStatus()
 	switch {
-	case wStat.GetName() == "":
```

Comment:
find fixing this possible panic: https://github.com/hashicorp/boundary/blob/88ede3f9e5d4e4d81d242871715878e6b5011f80/internal/daemon/cluster/handlers/worker_service.go#L227

Reply:
Thanks! I've updated the LookupSessionRequest to pass keyId/ added a lookup on that